src/styles/reset.less

```diff
@@ -92,6 +181,8 @@ label {
   margin: 0;
   padding: 0;
   vertical-align: baseline;
+  color: var(--theme-text-on-background-1);
+  transition: background 0.3s ease;
```

Comment:
may also want to transition color as well (and anywhere color is set), otherwise it can look kinda weird when toggling theme